common/changes/@itwin/core-frontend/nam-fix-appearance_2024-08-02-13-56.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-frontend",
+      "comment": "Add subcategory to appearance on default view creation if id doesn't exist in appearance map",
```

Comment:
Is this needed if this is due to a change in a previous pre-release and never seen by users?